src/main/java/com/cburch/logisim/fpga/hdlgenerator/HDL.java

```diff
@@ -55,6 +55,24 @@ public static String getRemarkStart() {
     return " ** ";
   }
 
+  public static String startIf(String condition) {
+    return isVHDL() ? LineBuffer.format("IF ({{1}}) THEN", condition)
+                    : LineBuffer.format("if ({{1}}) begin", condition);
+  }
+
+  public static String elseStatement() {
+    return isVHDL() ? "ELSE " : "end else begin";
+  }
+
+  public static String elseIf(String condition) {
+    return isVHDL() ? LineBuffer.format("ELSIF ({{1}}) THEN", condition)
+                    : LineBuffer.format("end else if ({{1}}) begin", condition);
+  }
+
+  public static String endIf() {
+    return isVHDL() ? "END IF;" : "end";
+  }
+
   public static String assignPreamble() {
     return isVHDL() ? "" : "assign ";
```

Comment:
Even it is not the part of this PR, it do no like traling space here. Can we drop it and amend all the code using it and add the space there?